internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -101,33 +102,45 @@ func NewApplicationSnapshotImage(ctx context.Context, fs afero.Fs, url string, p
 		os.Setenv("SIGSTORE_TRUST_REKOR_API_PUBLIC_KEY", "1")
 	}
 
-	policySources, err := fetchPolicySources(p)
-	if err != nil {
-		log.Debug("Failed to fetch the policy sources from the ECP!")
-		return nil, err
-	}
+	// Return an evaluator for each of these
+	for _, sourceGroup := range p.EnterpriseContractPolicySpec.Sources {
```

Comment:
One improvement, in a followup, could be that the work within this loop is done via in parallel via gorutines

Reply:
I added a comment to suggest that.